### ISSUE TITLE
Monorepo

### ISSUE THREAD

**Issue by jhen0409** [zalmoxisus/redux-devtools-extension#412]
I think we could maintaining packages like `remotedev-app`, `remotedev-utils`, `devui` and the [helper package](https://github.com/zalmoxisus/redux-devtools-extension/tree/master/npm-package) in monorepo, use [Yarn workspaces](https://yarnpkg.com/blog/2017/08/02/introducing-workspaces/) or [Lerna](https://github.com/lerna/lerna).

@zalmoxisus what do you think about this?

**Comment by zalmoxisus** [comment]
I was thinking about this long time ago. But we either keep everything (including `redux-devtools`, `react-native-debugger`, `remotedev-server` and probably monitors as well) in a monorepo or not. I think mostly contributors would want to change something in monitors then see it in the extension or `react-native-debugger`, which is a rather difficult process now.

**Comment by jhen0409** [comment]
Since some monitors are no positive maintaining, that would be great if we could move the monitors here.

For `react-native-debugger`, I think it may not easy to move here because most issues are not related to Redux DevTools, and we have to use [Releases](https://github.com/jhen0409/react-native-debugger/releases) for auto update. I might move the [implementation](https://github.com/jhen0409/react-native-debugger/blob/master/app/worker/reduxAPI.js) here and make a test app.

**Comment by zalmoxisus** [comment]
Agree, it would make it much easier to contribute. Would you like to implement it here? Then I'll move all the parts.

**Comment by jhen0409** [comment]
Yes, I'll implement it in a few days. :)

**Comment by zalmoxisus** [comment]
Thanks! :)
I'll make a new patch this days to fix some issues. Then we can start moving everything.

**Comment by zalmoxisus** [comment]
If anyone interested in this issue, please refer also to https://github.com/reduxjs/redux-devtools/issues/412
